pkg/config/config.go

```diff
@@ -559,3 +571,43 @@ func concatenateBytes(data ...[]byte) []byte {
 	}
 	return bytes
 }
+
+// AddOrgToConsortium adds an org definition to a named consortium in a given
+// channel configuration.
+func AddOrgToConsortium(org *Organization, consortium, channelID string, config *cb.Config, mspConfig *mb.MSPConfig) (*cb.ConfigUpdate, error) {
+	if org == nil {
+		return nil, errors.New("organization is empty")
+	}
+	if consortium == "" {
+		return nil, errors.New("consortium is empty")
+	}
+
```

Comment:
a nit: but IMO worth adding checks for rest of the parameters? Briefly inspecting the code flow nil config or mspConfig potentially could lead to a panic. (I know this cannot really happen, but would add this for sanity check).